packages/eds-core-react/src/components/Dialog/Dialog.docs.mdx

```diff
@@ -0,0 +1,75 @@
+import { Links, PropsTable, Story } from './../../../.storybook/components'
+import { Dialog } from '.'
+
+# Dialog
+
+Informs users about a task and can contain critical information, requires decisions, or involves multiple tasks, in a layer above the page.
+
+<Links 
+    ariaUrl="https://www.w3.org/TR/wai-aria-practices/#dialog_modal"
+    documentationUrl="https://eds.equinor.com/0b0c666ab/p/059c0e-dialog/b/087c3e"
+    sourceUrl="https://github.com/equinor/design-system/blob/develop/packages/eds-core-react/src/components/Dialog/Dialog.tsx"
+    npmUrl="https://www.npmjs.com/package/@equinor/eds-core-react"
+></Links>
+
+<Story id="feedback-dialog--introduction" />
+<PropsTable />
+
+## Usage
+
+Dialogs disable all app functionality when they appear, and remain on screen until confirmed, dismissed, or a required action has been taken.
+Dialogs are purposefully interruptive, so they should be used sparingly.
+
+<ul>
+    <li>Dialogs <strong>focus</strong> user attention to ensure their content is addressed.</li>
+    <li>Dialogs should be <strong>direct</strong> in communicating information and dedicated to completing a assignment.</li>
+    <li>Dialogs should appear in response to a user task or an action, with suitable or contextual information.</li>
+</ul>
+
+ ```tsx
+import { Dialog } from '@equinor/eds-core-react'
+
+<Dialog />
```

Comment:
A Dialog is not self-closing but requires children and has a set of sub-components, perhaps the usage example should include a more complete example?

Reply:
Yeah, i did find out that to, but wanted to wait for someone to review it, and maybe find some more bugs 😅